internal/service/advancedclustertpf/testdata/TestMockAdvancedCluster_configSharded.yaml

```diff
@@ -0,0 +1,474 @@
+step_count: 3
```

Comment:
is step_count really needed? it seems to be the number of diff_requests

Reply:
Not really needed, however, I like it for readability

Reply:
fair enough, in that case I would check that step_count and the counts are the same or raise an error